packages/dds/sequence/src/sequenceDeltaEvent.ts

```diff
@@ -73,7 +76,10 @@ export abstract class SequenceEvent<
 
 	/**
 	 * The in-order ranges affected by this delta.
-	 * These may not be continuous.
+	 * These are not necessarily continuous.
```

Comment:
I'm not exactly sure what this meant to express, but maybe "contiguous" would be clearer.
```suggestion
	 * These are not necessarily contiguous.
```

Reply:
Contiguous is more accurate; intent is that under certain circumstances, the range of segments that the delta affects aren't a single range. For example, if:

1. Local client removes a range of text
2. Remote client inserts text in the middle of that range

At the time the removal gets acked on the local client, the maintenance event that gets raised will contain segments which are separated by the newly inserted text (since the remote client splits the removed range into more than one part). There are other similar sorts of scenarios where this thing can occur (annotating a range rather than removing it, for example)